pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -61,8 +61,31 @@ const (
 
 	operationPollInterval        = 3 * time.Second
 	operationPollTimeoutDuration = 30 * time.Minute
+
+	defaultLBSourceRange = "0.0.0.0/0"
+
+	//Expected annotations for GCE
+	gceLBAllowSourceRange = "kubernetes.io/gce.loadbalancer.allow.source.range"
```

Comment:
1. can you look at other annotations formats that do the same thing? I think i saw some 'kubernetes.io/aws-.*'
2. can you look if there's a "net.alpha" somewhere, if so, we should probably classify this as "beta"
3. please use a single word instead of loadbalancer.allow.source.range

@bgrant0607 may be able to comment on the format of a beta feature that's cloud provider specific, that we're surfacing through annotations.